doc/classes/ParticleProcessMaterial.xml

```diff
@@ -405,6 +405,13 @@
 			A pivot point used to calculate radial and orbital velocity of particles.
 		</member>
 	</members>
+	<signals>
+		<signal name="emission_shape_changed">
+			<description>
+				Emitted when the emission shape of the material has been changed in its offset, size, scale, or orientation.
```

Comment:
I would directly mention _some_ of the properties at interest. For example:
```suggestion
				Emitted when this material's emission shape is changed in any way. This includes changes to [member emission_shape], [member emission_shape_scale], or [member emission_sphere_radius], and any other property that affects the emission shape's offset, size, scale, or orientation.
```
Not sure which is most relevant (perhaps `emission_sphere_radius` should be replaced by `emission_shape_offset`?).
I'd at least keep one that is not prefixed by `emission_shape` just to show that not all of these related properties contain the word "_shape_". I'll let you be the judge.